### PR TITLE
Printing error stack and message when thrown

### LINKED ISSUE #25: Error not being logged due to cloning
## Problem:
When using a logger from this module if we supply an error it is not logged
So example:
```js
logger.error('Oooups!', new Error('test'));
```

would return:
`error: Opus`

Expected:
`error: Opus, Error: stack:..., message:' test'`

So error is swallowed.

The problem is in this line
https://github.com/dial-once/node-microservice-boot/blob/develop/src/winston.js#L26

if meta is instance of Error Object.assign will not do anything on it and return {} ergo the error is lost.

### PR DESCRIPTION
**Pull request by Spring3**
resolves #25

### CODE REVIEW

File: src/winston.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ module.exports = (notifier, token, notify = true) => {
       logger = loggers[token];
     }
     logger.rewriters.push((level, msg, meta) => {
-      const newMeta = Object.assign({}, meta);
+      const newMeta = Object.assign({}, meta instanceof Error ? { error: meta.stack, message: meta.message } : meta);
```
Comment (mrister) [inline src/winston.js]: I think error: meta.stack should be stack: meta.stack

File: test/winston-spec.js [hunk 2]
```diff
@@ -114,10 +112,15 @@ describe('winston transports', () => {
     mock.restore();
   });
 
+  it('should print the error stack as well as message if error occurred', () => {
+    const logger = require(modulePath)('');
+    const error = new Error('Test error');
+    logger.error('Oooups!', error);
```
Comment (mrister) [inline test/winston-spec.js]: tests need t otest something

File: test/winston-spec.js [hunk 3]
```diff
@@ -114,10 +112,22 @@ describe('winston transports', () => {
...
+  it('should print the error stack as well as message if error occurred', () => {
+    const logger = require(modulePath)(sinon.stub(), '00000000-0000-0000-0000-000000000000', false);
+    const error = new Error('Test error');
+    const spy = sinon.spy(logger.rewriters[logger.rewriters.length - 1]);
+    logger.rewriters[logger.rewriters.length - 1] = spy;
+    logger.error('Oooups!', error);
+    assert(spy.called);
+    assert(spy.calledWith('error', 'Oooups!', error));
+    const returnValue = spy.returnValues[0];
+    assert({}.hasOwnProperty.call(returnValue, 'message'));
```
Comment (mrister) [inline test/winston-spec.js]: You could do Object.hasOwnProperty but it's ok

**Comment by ky23** [comment]
@Spring3 Please fix tests before merge

**Pull request merged**